lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -406,9 +406,9 @@ def test_enroll_member_student(self):
         self.assertEqual(response.status_code, 200)
         # we were redirected to our current location
         self.assertEqual(len(response.redirect_chain), 1)
-        self.assertTrue(302 in response.redirect_chain[0])
+        self.assertIn(302, response.redirect_chain[0])
         self.assertEqual(len(outbox), 1)
-        self.assertTrue(student.email in outbox[0].recipients())  # pylint: disable=no-member
+        self.assertIn(student.email, outbox[0].recipients())  # pylint: disable=no-member
```

Comment:
If I remove it I get this error:
```
lms/djangoapps/ccx/tests/test_views.py:411: [E1101(no-member), TestCoachDashboard.test_enroll_member_student] Instance of 'SubFactory' has no 'email' member
```

I think it does not see the `email` property of the object.